src/sbd-inquisitor.c

```diff
@@ -923,12 +926,18 @@ int main(int argc, char **argv, char **envp)
                 }
             }
         }
+
+        value = getenv("SBD_REBOOT_ACTION");
+        if(value) {
+            reboot_action = strdup(value);
+        }
+
```

Comment:
Not a big deal, but since the explicit option takes precedence, I'd move this after the option parsing and only check it if it wasn't explicitly set

Reply:
yes, in general this saves some code to be executed and implementing from scratch I would probably opt for doing it this way round.
here I had decided to first check the environment-variable and overwrite it after as this logic is already applied for the other environment-variables. thought consistency here would be good for readability of the code. And the purge&replace implementation is as well an easy way to handle multiple occurrences while probably moaning would be more user-friendly (Concatenation would be another possibility I thought of but given we don't have flags that can't independently be set but rather 2 fields with 2 states one and 3 states the other I had the impression that would render things even more obfuscated.).

Reply:
that makes sense too